docs/platforms/apple/common/features/experimental-features.mdx

```diff
@@ -34,9 +38,7 @@ SentrySDK.start { options in
 }];
 ```
 
-## Stitch Together Swift Concurrency Stack Traces
-
-<Note>This feature is experimental and may have bugs.</Note>
+#### Stitch Together Swift Concurrency Stack Traces
```

Comment:
this was intentional so that it doesn't create a (the only) menu item in the right sidebar. this will most likely change when we iterate over the Experimental Features page :)

Reply:
Ah ok. Got it. The only concern about this is that it may create confusion for people using e-readers since the e-readers uses the paragraph headings to understand page hierarchy. Maybe we just bold it instead of adding any ##?